client/src/document/organisms/article-footer/index.tsx

```diff
@@ -28,6 +31,62 @@ export function LastModified({ value, locale }) {
   );
 }
 
+export function Contributors({ url }) {
+  const contributorJSONUrl = `https://api.github.com/repos/mdn/translated-content/commits?path=files/${url}/index.md`;
+
+  const { error, data = [] } = useSWR<any>(contributorJSONUrl, async (url) => {
+    const response = await fetch(url);
```

Comment:
I think we should provide GitHub Token when available: https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication#example-2-calling-the-rest-api

Ref: [Rate limits for the REST API](https://docs.github.com/en/rest/using-the-rest-api/rate-limits-for-the-rest-api?apiVersion=2022-11-28)